### PR TITLE
Changed Zeit.co to Vercel due to its recent name change.

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
## Description

Changed Zeit.co to Vercel due to its recent name change.

## Checklist

- [x] I’ve added tests to confirm my change works.
- [x] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Review by sosukesuzuki — COMMENTED** [review]
File changes are look good to me. However can you clean git history on this PR? Some extra commits included.

**Comment by Harry-Hopkinson** [comment]
Those are some commits on the other branches. How would I remove extra commits?

File: tests/format/markdown/markdown/real-world-case.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ A few of the [many projects](https://www.npmjs.com/browse/depended/prettier) usi
 </tr>
 <tr>
 <td><p align="center"><a href="https://babeljs.io/"><img src="website/static/images/babel-200x100.png" alt="Babel" width="200" height="100"><br>Babel</a></p></td>
-<td><p align="center"><a href="https://zeit.co/"><img src="website/static/images/zeit-200x100.png" alt="Zeit" width="200" height="100"><br>Zeit</a></p></td>
+<td><p align="center"><a href="https://vercel.com/"><img src="website/static/images/zeit-200x100.png" alt="Vercel" width="200" height="100"><br>Vercel</a></p></td>
```
Comment (fisker) [inline tests/format/markdown/markdown/real-world-case.md:183]: Please revert changes in the `tests/` this don't need sync with the website.

File: website/data/users.yml [hunk 2]
```diff
@@ -43,10 +43,10 @@
   greyImage: /images/users/used_by_storybook.svg
   infoLink: https://storybook.js.org/
   pinned: true
-- caption: ZEIT
+- caption: Vercel
   image: /images/users/zeit-200x100.png
```
Comment (fisker) [inline website/data/users.yml:47]: Can you also update the logo? We can use the svg from their website.

**Pull request closed without merging**